src/test/ui/repr/rustfix.stderr

```diff
@@ -0,0 +1,162 @@
+error[E0693]: incorrect `repr(align)` attribute format
+  --> $DIR/rustfix.rs:2:8
+   |
+LL | #[repr(align = r#"1"#)]
+   |        ^^^^^^^^^^^^^^
+   |
+help: use parentheses instead
+   |
+LL | #[repr(align(1))]
+   |             ^ ^
+
+error[E0693]: incorrect `repr(align)` attribute format
+  --> $DIR/rustfix.rs:6:8
+   |
+LL | #[repr(align = r###"foo"###)]
+   |        ^^^^^^^^------------
+   |                |
+   |                must be a non-negative number
+   |
+help: use parentheses instead
+   |
+LL | #[repr(align(foo))]
+   |             ^   ^
```

Comment:
Should this be divided into two separate errors or is it fine left as is?

Reply:
It's somewhat concerning that it suggests "use parentheses" and leaving `foo` alone when a number is expected there. This might not need to be addressed now.

> Should this be divided into two separate errors or is it fine left as is?

What do you mean? Having one for the lack of parentheses and a separate for `foo` not being a positive number? Personally I prefer to consolidate errors when possible, and I think it's fine overall as is.